kubernetes/linux/setup.sh

```diff
@@ -39,7 +39,7 @@ mv /usr/lib/ruby/gems/3.1.0/specifications/default/uri-0.11.0.gemspec /usr/lib/r
 gem uninstall time --version 0.2.0
 gem uninstall uri --version 0.11.0
 
-sudo tdnf install -y azure-mdsd-1.27.4
+sudo tdnf install -y azure-mdsd-1.28.11
```

Comment:
Can you please check the mdsd logs and see if you are not seeing any errors in the new version? and also, can you please check no regressions in syslog?  Once we get this change to CI/CD clusters, we would need to watch resource usage and data flow, and errors. Please have a task/plan to validate MDSD update related changes.